ckcp/openshift_dev_setup.sh

```diff
@@ -34,6 +34,14 @@ Optional arguments:
         Directory in which to create the gitops file structure.
         If the directory already exists, all content will be removed.
         By default a temporary directory will be created in $TMPDIR.
+    --tekton-results-database-user TEKTON_RESULTS_DATABASE_USER
```

Comment:
@CathalOConnorRH Can you please move the database params in `config/config.yaml`.
In "openshift_dev_setup.sh" we only deal with global params, but for specific app configs we centralise that in "config/config.yaml" otherwise we may end up with a myriad of script args for the different apps to configure.
Smth like the following would work:
```yaml
tekton_results_db:
  user:
  password:
```
You can find in the ckcp script how we parse that file.